hci.md

```diff
@@ -558,6 +521,18 @@ for the hypervisor. In an IOPS-optimized cluster performance can be
 improved by reserving more memory per OSD.  The 5 GB number is
 provided as a starting point which can be further tuned if necessary.
 
-The `$FSID` value above should contain the actual FSID as described
-in the "Create a Ceph Secret" section of the [documentation to
-configure OpenStack to use Ceph](ceph.md).
+Use `oc edit OpenStackDataPlaneService/nova-custom-ceph` to add
+`resereved-memory-nova` to the `configMaps` list.
+
+```yaml
+---
+kind: OpenStackDataPlaneService
+<...>
+spec:
+  configMaps:
+  - ceph-nova
```

Comment:
@fultonj not sure I get why we can't put the two contents (`03-ceph-nova.conf` and `04-reserved-memory-nova.conf`) within a single `ConfigMap` instead of having both ceph-nova` and `reserved-memory-nova`

Reply:
We could combine them and either results in the same configuration. I separated them because:

- 03-ceph-nova.conf is from the ceph document and needed for external Ceph only
- 04-reserved-memory-nova.conf is from the HCI document and needed only for the 

Combining them  would not let me reference the first and have to document twice. Either one produces a valid config. One could argue it helps organize logically separate configurations.